packages/lab/src/pagination/Pagination.tsx

```diff
@@ -16,27 +16,37 @@ import { withBaseName } from "./utils";
 const { Provider } = paginationContext;
 
 export interface PaginationProps extends HTMLAttributes<HTMLElement> {
+  /**
+   * Number of pages.
+   */
   count: number;
+  /**
+   * Current/active page.
+   */
   page?: number;
+  /**
+   * Default current/active page.
+   */
   initialPage?: number;
+  /**
+   * Callback function triggered when current page changed.
+   */
   onPageChange?: (page: number) => void;
-  compact?: boolean;
 }
 
 export const Pagination = forwardRef<HTMLElement, PaginationProps>(
```

Comment:
Is there a more specific ref type we can use here?

Reply:
none that I can think of for <nav>